### PR TITLE
remove_Explorer.openTableSelectQueryPanel

### PR DESCRIPTION
**Pull request by hardiknai-techm**
[Preview this branch](https://cosmos-explorer-preview.azurewebsites.net/pull/881?feature.someFeatureFlagYouMightNeed=true)

### CODE REVIEW

File: src/Explorer/Tables/QueryBuilder/QueryViewModel.tsx [hunk 1]
```diff
@@ -111,7 +113,7 @@ export default class QueryViewModel {
     DataTableUtilities.forceRecalculateTableSize(); // Fix for 261924, forces the resize event so DataTableBindingManager will redo the calculation on table size.
   };
 
-  public ontoggleAdvancedOptionsKeyDown = (source: any, event: KeyboardEvent): boolean => {
```
Comment (southpolesteve) [inline src/Explorer/Tables/QueryBuilder/QueryViewModel.tsx:114]: Please verify this is not breaking. If something is calling this method then the parameter order changed

File: src/Explorer/Tables/QueryBuilder/QueryViewModel.tsx [hunk 2]
```diff
@@ -197,12 +197,11 @@ export default class QueryViewModel {
...
 
-  public selectQueryOptions(): Promise<any> {
-    this.queryTablesTab.container.openTableSelectQueryPanel(this);
-    return null;
+  public selectQueryOptions() {
+    useSidePanel.getState().openSidePanel("Select Column", <TableQuerySelectPanel queryViewModel={this} />);
   }
 
-  public onselectQueryOptionsKeyDown = (source: any, event: KeyboardEvent): boolean => {
+  public onselectQueryOptionsKeyDown = (event: KeyboardEvent): boolean => {
```
Comment (southpolesteve) [inline src/Explorer/Tables/QueryBuilder/QueryViewModel.tsx:204]: Same as above

Comment (hardiknai-techm) [reply src/Explorer/Tables/QueryBuilder/QueryViewModel.tsx]: done.
https://microsoft.github.io/PowerBI-JavaScript/interfaces/_node_modules_typedoc_node_modules_typescript_lib_lib_dom_d_.keyboardevent.html

File: src/Explorer/Tables/QueryBuilder/QueryViewModel.tsx (hunk 1 above)
Comment (hardiknai-techm) [reply src/Explorer/Tables/QueryBuilder/QueryViewModel.tsx:114]: done.
https://microsoft.github.io/PowerBI-JavaScript/interfaces/_node_modules_typedoc_node_modules_typescript_lib_lib_dom_d_.keyboardevent.html

**Pull request merged**